frontend/lib/identification/activation_workflow/activation_code_parser.dart

```diff
@@ -22,7 +24,7 @@ class QRCodeInvalidFormatException extends QrCodeParseException {
 class ActivationCodeParser {
   const ActivationCodeParser();
 
-  DynamicActivationCode parseQrCodeContent(String rawBase64Content) {
+  DynamicActivationCode parseQrCodeContent(Uint8List rawBase64Content) {
     final QrCode qrCode = rawBase64Content.parseQRCodeContent();
```

Comment:
I guess the variable should not be called `rawBase64Content` anymore?

Reply:
Thanks, I'll rename!